.github/workflows/CI-lint-format.yml

```diff
@@ -17,6 +17,7 @@ env:
   DOCKER_CARGO_HOME: /tmp/.cargo
   CMAKE_INSTALL: true
   LLD_INSTALL: true
+  CARGO_BINARIES_INSTALL: "zepter"
```

Comment:
Considering zepter doesn't require any build, perhaps it could be worth it to move it to a separate job ?